api/api/services/internal_tasks/meta_agent_service.py

```diff
@@ -419,13 +420,28 @@ async def _build_meta_agent_input(
         current_agent: SerializableTaskVariant,
         playground_state: PlaygroundState,
     ) -> tuple[MetaAgentInput, list[SerializableTaskRun]]:
-        # Concurrently extract company info and list current agents
-        company_description, existing_agents, agent_runs = await asyncio.gather(
+        (
+            company_description,
+            existing_agents,
+            agent_runs,
+            feedback_page,
+            has_active_runs,
+            reviewed_input_count,
+        ) = await asyncio.gather(
             safe_generate_company_description_from_email(user_email),
             list_agent_summaries(self.storage, limit=10),
             self.fetch_agent_runs(task_tuple, playground_state.agent_run_ids),
+            self.feedback_service.list_feedback(
+                task_tuple[1],
+                run_id=None,
+                limit=10,
+                offset=0,
+                map_fn=MetaAgentInput.AgentLifecycleInfo.FeedbackInfo.AgentFeedback.from_domain,
+            ),
+            self.has_active_agent_runs(task_tuple, agent_schema_id),
+            self.get_reviewed_input_count(task_tuple, agent_schema_id),
```

Comment:
`return_exception=True` ? 

Otherwise you might end up with all call failing when a single one fails

Reply:
@guillaq I've implemented a safer context fetching in this [commit](https://github.com/WorkflowAI/WorkflowAI/pull/29/commits/0b9dbff36ed6c935d0391e6a2588454f09484718). Do you want to check again? 

thanks !